### PR TITLE
Global style guide

### PR DESCRIPTION
**Pull request by micahtaylor**
Covers the following:
https://trello.com/c/gDqJHl32/404-attachment-display-in-current-app
https://trello.com/c/lT1co6xF/504-drop-downs-display-incorrectly
https://trello.com/c/PClLTEWC/502-match-notification-bar-and-text-colors-to-design-file

### CODE REVIEW

File: app/assets/stylesheets/redesign/_card-action.scss [hunk 1]
```diff
@@ -15,16 +15,26 @@ ul.request-actions{
     outline: none;
   }
   .save-button .button{
-    border-color: #8ecc76;
-    background: #8ecc76;
+    border-color: map-get($c2success-swatch, primary);
```
Comment (rememberlenny) [inline app/assets/stylesheets/redesign/_card-action.scss]: Thanks for implementing this. Long overdue.

File: app/assets/stylesheets/redesign/_global.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
+// Colors
+$c2blue-swatch: (
```
Comment (rememberlenny) [inline app/assets/stylesheets/redesign/_global.scss]: Did you see the variable file? @micahsaul 

https://github.com/18F/C2/blob/master/app/assets/stylesheets/redesign/_foundation_settings.scss

File: app/views/proposals/details/_attachment.html.haml [hunk 3]
```diff
@@ -16,3 +14,5 @@
           .medium-12.column
             = render partial: "proposals/details/attachment_list",
               locals: { proposal: @proposal }
+            - if !@proposal.completed? && !@proposal.canceled?
```
Comment (rememberlenny) [inline app/views/proposals/details/_attachment.html.haml]: I think @nickbristow just made a change to this line. Going to follow up to make sure you two dont overwrite each other.

Comment (rememberlenny) [inline app/views/proposals/details/_attachment.html.haml]: Yep. He did it here: https://github.com/18F/C2/pull/1321

Before merging, can you make sure your branch is up to date? @micahsaul

**Comment by rememberlenny** [comment]
Made comments! @micahsaul

**Comment by micahsaul** [comment]
@rememberlenny I think you mean @micahtaylor.

😄

**Comment by micahtaylor** [comment]
Updated for https://trello.com/c/gDqJHl32/404-attachment-display-in-current-app and made sure this branch is in step with master.
Back to you @rememberlenny

**Comment by rememberlenny** [comment]
Looks good!

**Pull request merged**